index.js

```diff
@@ -47,7 +47,7 @@ module.exports = class Autobase extends ReadyResource {
     this.keyPair = handlers.keyPair || null
     this.valueEncoding = c.from(handlers.valueEncoding || 'binary')
     this.store = store
-
+    this.globalCache = store.globalCache || null
```

Comment:
It's exposing `this.globalCache` for convenience (easier to know if the autobase is using it), but it's not strictly needed. 

The `globalCache` is only used in the autocore-sessions, which in theory could also get it directly with `this.base.store.globalCache`.